demos/demo/src/modules/user/index.js

```diff
@@ -1,39 +1,45 @@
 import {set} from 'cerebral/operators'
 import {props, state} from 'cerebral/tags'
-import {form, changeField} from 'cerebral-forms'
 import createUser from './signals/createUser'
 import signIn from './signals/signIn'
 import signInAnonymous from './signals/signInAnonymous'
 import signOut from './signals/signOut'
+import {setField} from 'cerebral-provider-forms/operators'
 
 export default {
   state: {
     lang: 'en',
     $loggedIn: false,
-    $signIn: form({
+    $signIn: {
       email: {
         value: '',
         validationRules: ['isEmail'],
-        validationMessages: ['loginValidationEmailNotValid'],
+        validationMessages: {
+          'isEmail': 'loginValidationEmailNotValid'
+        },
         isRequired: true,
         requiredMessage: 'loginValidationEmailRequired'
       },
       password: {
         value: '',
         validationRules: ['minLength:5'],
-        validationMessages: ['loginValidationPasswordTooShort'],
+        validationMessages: {
+          'minLength': 'loginValidationPasswordTooShort'
+        },
         isRequired: true,
         requiredMessage: 'loginValidationPasswordRequired'
```

Comment:
So in cerebral-forms the behavior was that if a required field was empty the requiredMessage was shown. This is not the case anymore. 

BTW I was talking about a bug in cerebral-provider-forms not the demo.